src/content/guides/pay-payment-app-integration.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Paying a Payment Request
+description: Integrate your app with Centrapay to pay payment requests on merchant terminals and POS
+nav:
+  path: Reference/App Integrations
+  order: 1
+---
+
+## Overview
+
+This guide is for a [Third-party Asset Provider](/guides/integrating-third-party-asset) to enable paying with their
+digital assets via Centrapay.
+
+## Authenticating API calls on behalf of users
+
+Third-party applications can make API calls on behalf of users with Centrapay
+API keys. All API requests to Centrapay must be made from the integrator’s backend.
+Centrapay API keys must also be managed on the backend and stored as secrets to
+minimize the risk of API keys being compromised.
+
```

Comment:
Do we want to add a comment here about rotating API keys?

Reply:
Do you have a suggestion on what to say? Maybe that's another general guide we should make in the future

Reply:
I think we should have a Managing API Keys guide. Rotating API Keys is important but I don't know that it needs to be mentioned here. Should be part of the certification process though.